docs/manual_api/core.rst

```diff
@@ -1,3 +1,43 @@
+====
 Core
 ====
 
+PlaidML Core
+
+.. contents::
+
+--------------
+Initialization
+--------------
+
+.. tabs::
+
+   .. group-tab:: C++
+
+      .. doxygenfunction:: plaidml::init
+
+   .. group-tab:: Python
+
+      .. autofunction:: plaidml.core.__init
```

Comment:
I think we should avoid adding private impl details into docs. (In python, anything that starts with `__` is private, and `_` is semi-private)

Reply:
There isn't any correlated initialization in python, it happens when the module gets imported. We could mention that in this section, but not actually point to `__init` which is private an may change at any time